cypress/integration/login.spec.js

```diff
@@ -53,15 +53,27 @@ describe('Login', () => {
     it('alerts on invalid user', () => {
       cy.login('invalid@user.com', password, useCachedSession)
 
-      cy.contains('.alert-danger', 'Login ou senha incorretos')
+      cy.contains('.alert-danger', 'Incorrect login or password')
         .should('be.visible')
     })
 
     it('alerts on invalid password', () => {
       cy.login(Cypress.env('user'), 'invalid-pwd', useCachedSession)
 
-      cy.contains('.alert-danger', 'Login ou senha incorretos')
+      cy.contains('.alert-danger', 'Incorrect login or password')
+        .should('be.visible')
+    })
+
+    it('change language', () => {
```

Comment:
@wlsf82 Can you validate if everything is ok with this test?

Reply:
@ArthurMota9, it looks good to me.
Did you run them? Are they still passing?

By the way, soon, I intend to work on having them running on CI.

Reply:
All tests passed, I just mentioned you to verify that the code respects the standard. Thanks!